nicegui/elements/echart.js

```diff
@@ -1,9 +1,26 @@
+import { convertDynamicProperties } from "../../static/utils/dynamic_properties.js";
+
 export default {
   template: "<div></div>",
   mounted() {
     this.chart = echarts.init(this.$el);
+    convertDynamicProperties(this.options, true);
     this.chart.setOption(this.options);
     this.chart.resize();
+    function unpack(e) {
```

Comment:
Why have you picked this subset of  event arguments? Have you seen the `uncycle` function in `chart.js`?

Reply:
From what I could see in the documentation these are the options available in the response from the ```click``` event.
I saw the ```uncycle``` in ```chart.js``` but I'm not great with javascript and didn't understand it.

Reply:
Ok, we will let @falkoschindler decide which variant we will integrate. Thanks!